lib/moon/design/tabs/list.ex

```diff
@@ -20,6 +20,7 @@ defmodule Moon.Design.Tabs.List do
   prop(on_change, :event, from_context: :on_change)
 
   slot(tabs)
+  slot(default)
```

Comment:
also add some documentation here. things to be explained: tabindex & is_selected should be set manually

Reply:
No it should not. It works nicely. Please test.